package.json

```diff
@@ -3,12 +3,19 @@
   "version": "0.23.3",
   "description": "Connexions (cnx.org) web site",
   "scripts": {
+    "precommit": "lint-staged",
```

Comment:
can you include linting into the `posttest` command? That way Travis will run it too! Maybe `"posttest": "lint-staged && node ./script/check-templates-for-non-i18n-text.js",`

Reply:
`npm run precommit` would work for staged file or `stylelint path/to/the/file/you/want/to/lint`
`stylelint path/to/the/file --syntax=less --custom-formatter=node_modules/stylelint-formatter-pretty` would be even better.